src/ErrorBoundary.jsx

```diff
@@ -25,6 +25,7 @@ class ErrorBoundary extends Component {
       return (
         <>
           <Header />
+          <Footer />
```

Comment:
Hey Ermias, am I correct that the error should show up *between* `Header` and `Footer` rather than below them?